src/js/menu.js

```diff
@@ -101,7 +95,7 @@ var showSensorsMenu = function(thermostat) {
     }
   });
 
-  menu = new UI.Menu({
+  var sensorMenu = new UI.Menu({
```

Comment:
I have mixed feelings about that. From the point of how thermostat menu works and how all mobile apps work, this is correct. At the same time we are dealing with a watch, and we might want to minimize number of clicks. Especially when average user accesses a menu to do only one action: remove hold or see sensor values. 
So let's use your approach and see how users react. I bet noone will complain.